pkg/homedir/homedir.go

```diff
@@ -5,14 +5,20 @@ import (
 	"runtime"
 )
 
+// Return the Env Var Name for the user's home dir. On Linux this is
+// "HOME" while on windows its "USERPROFILE"
```

Comment:
this should be starting with function's name per idiomatic Go.

> HomeKey returns the environment name for user's home directory
> depending on platform running on. 

is probably just fine. By adding more stuff you're risking comments to get out of sync with code. :)